ArduCopter/control_guided.cpp

```diff
@@ -579,13 +586,23 @@ void Copter::guided_angle_control_run()
         roll_in = 0.0f;
         pitch_in = 0.0f;
         climb_rate_cms = 0.0f;
+        yaw_rate_in = 0.0f;
     }
 
     // set motors to full range
     motors.set_desired_spool_state(AP_Motors::DESIRED_THROTTLE_UNLIMITED);
 
     // call attitude controller
-    attitude_control.input_euler_angle_roll_pitch_yaw(roll_in, pitch_in, yaw_in, true, get_smoothing_gain());
+    if(guided_angle_state.use_yaw_rate)
+        attitude_control.input_euler_angle_roll_pitch_euler_rate_yaw(roll_in, pitch_in, yaw_rate_in, get_smoothing_gain());
+    else
+        attitude_control.input_euler_angle_roll_pitch_yaw(roll_in, pitch_in, yaw_in, true, get_smoothing_gain());
+
+    // call throttle controller
+    if (rangefinder_alt_ok()) {
```

Comment:
You're right, its a leftover from some other changes I made that I didn't catch.